resource-managers/kubernetes/integration-tests/scripts/setup-integration-test-env.sh

```diff
@@ -106,7 +106,11 @@ then
     # OpenJDK base-image tag (e.g. 8-jre-slim, 11-jre-slim)
     JAVA_IMAGE_TAG_BUILD_ARG="-b java_image_tag=$JAVA_IMAGE_TAG"
   else
-    JAVA_IMAGE_TAG_BUILD_ARG="-f $DOCKER_FILE_BASE_PATH$DOCKER_FILE"
+    if [[ $DOCKER_FILE = /* ]]; then
```

Comment:
another way could be to use `if [[ -f $DOCKER_FILE ]]`, i.e. to check that the file exists or not

Reply:
Thanks. Check Dockerfile exists or not when building the image. I don't think it's necessary to double check.